host/xtest/regression_1000.c

```diff
@@ -1917,6 +1917,11 @@ static void xtest_tee_test_1025(ADBG_Case_t *c)
 	uint32_t ret_orig = 0;
 	uint8_t *empty_buf = NULL;
 
+	if (!xtest_teec_ctx.memref_null) {
```

Comment:
This field is implementation defined and may be changed.
We could have a helper function/macro in `tee_client_api.h` to report this status instead. It is an extension and probably only useful here in xtest, but it will allow us to make changes to `TEEC_Context` without synchronizing with xtest.